pyproject.toml

```diff
@@ -27,7 +27,14 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-  "bioio-base>=0.2.0",
+  "bioio-base>=0.3.0",
+  "dask[array]>=2021.4.1",
+  "fsspec>=2022.8.0",
+  "numpy>=1.16",
```

Comment:
I think this minimum version doesn't support our minimum python version (3.9), `1.21.0` I think is the minimum